src/config.ts

```diff
@@ -355,6 +355,7 @@ export class Config implements IConfig {
     const command = c.load()
     await this.runHook('prerun', {Command: command, argv})
     await command.run(argv, this)
+    await this.runHook('postrun', {Command: command, argv})
```

Comment:
We should capture the results of the command and pass that into the port run hook. I would imagine a lot of post hooks would want to try and do something with the command results.